### DISCUSSION TITLE
news/20241115/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#418] [Website Discussions]
# news/20241115/

Today marks the third and final day of the Final Cut Pro Creative Summit- and what an incredible week it's been!

https://fcp.cafe/news/20241115/

**Comment by florian-duffe** [comment]
so great. thanks for the coverage, Chris. I wished there was a similar event in Europe. 
So many interesting reveals this year! Eddie AI, Jumper, Amove, Multicam Flattener, ... - at least two of them will be implemented into my workflow.
